src/pages/content/index.tsx

```diff
@@ -69,8 +69,26 @@ async function initialize() {
 
         if (pageData.type === "root") {
```

Comment:
Check code after this if statement

there are some extra navigation things and other stuff, so you should move your code below them

also, try to have only one `root.render()`, so that it doesn't re-render all